rssalertbot/alerts.py

```diff
@@ -20,16 +33,26 @@ def alert_email(feed, cfg, entry):
         cfg (dict):              output config
         entry (dict):            the feed entry
     """
-    log.debug(f"Alerting email: {entry.title}")
+    logger = logging.LoggerAdapter(log, extra = {
+        'feed':  feed.name,
+        'group': feed.group['name'],
+    })
+    logger.setLevel(logging.DEBUG)
```

Comment:
Looks like there are quite a few places where log level is manually set to debug throughout this PR.